mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -523,10 +588,13 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD s isPartOf (Laij;)Z
 		ARG 1 entity
 	METHOD t isSpectator ()Z
-	METHOD t handlePlayerAttack (Laij;)Z
+	METHOD t onAttack (Laij;)Z
+		ARG 1 attacker
 	METHOD u copyPositionAndRotation (Laij;)V
 		ARG 1 entity
 	METHOD u_ initDataTracker ()V
+	METHOD v initNewDimension (Laij;)V
```

Comment:
At first glance, with this name, I would assume that this method initializes a new dimension. In fact, it copies attributes from the original entity in the context of a dimension change. I think the name should reflect that, by containing "copy" or equivalent.

Reply:
A previous version used `copyFrom` for this method, however I noted that it seems to specifically exclude dimension information when copying, so it doesn't appear to be a generic copy function.

Reply:
Decided to go back to the `copyFrom` idea after reviewing its usages and overrides. (Don't see any places where it gets overriden)